atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/ProfilingCassandraClientTest.java

```diff
@@ -89,14 +90,16 @@ public void handlesNullMultigetSliceResponseFromDelegate() throws TException {
         List<ColumnOrSuperColumn> columns =
                 ImmutableList.of(new ColumnOrSuperColumn().setColumn(new Column(byteBuffer)));
         ImmutableMap<ByteBuffer, List<ColumnOrSuperColumn>> resultMap = ImmutableMap.of(byteBuffer, columns);
+        SlicePredicate slicePredicate = new SlicePredicate();
+        slicePredicate.setSlice_range(new SliceRange());
```

Comment:
seeing as this just goes into a mock that matches `any()`, we probably don't need this

Reply:
although, having said that, I appreciate the consistency, and it's definitely nicer doing a `verify()` on the same object that we passed in, instead of creating a new empty slice predicate twice

Reply:
Because this class previously just tested the mock, which was not the intention, and now actually tests the `profilingClient`, a NPE is thrown if `SliceRange` is empty

Reply:
aah, that makes sense - the previous test was definitely jank